packages/hoppscotch-common/src/components/smart/EnvInput.vue

```diff
@@ -176,10 +178,15 @@ const handleKeystroke = (ev: KeyboardEvent) => {
     return
   }
 
-  showSuggestionPopover.value = true
+  // show suggestions only when the user types a character other than enter or tab
+  if (ev.key === "Tab") {
```

Comment:
Can you make this into one line where if it is Enter, make the suggestions false?